alacritty/src/config/monitor.rs

```diff
@@ -69,26 +70,20 @@ pub fn watch(mut paths: Vec<PathBuf>, event_proxy: EventLoopProxy<Event>) {
         }
 
         loop {
-            let event = match rx.recv() {
-                Ok(event) => event,
+            let res = match rx.recv() {
+                Ok(res) => res,
                 Err(err) => {
                     debug!("Config watcher channel dropped unexpectedly: {}", err);
                     break;
                 },
             };
 
-            match event {
-                DebouncedEvent::Rename(_, path)
-                | DebouncedEvent::Write(path)
-                | DebouncedEvent::Create(path)
-                | DebouncedEvent::Chmod(path)
-                    if paths.contains(&path) =>
-                {
+            if let Ok(events) = res {
```

Comment:
Why are you ignoring this error completely?

Reply:
Because that's what the code was doing before, too.

Reply:
The code before didn't have a result here.

Reply:
The error was encoded in the [DebouncedEvent](https://docs.rs/notify/4.0.17/notify/enum.DebouncedEvent.html#variant.Error).  With the new API, they made it [an actual Result](https://docs.rs/notify-debouncer-mini/0.2.1/notify_debouncer_mini/type.DebounceEventResult.html).